backend/server/calculator.py

```diff
@@ -8,17 +8,23 @@ def calculate_fixed_rate(credit_amount, loan_term, interest_rate):
         MONTHS_IN_YEAR
         * (1 - (MONTHS_IN_YEAR / (MONTHS_IN_YEAR + interest_rate)) ** loan_term_months)
     )
-    base_monthly_payment = round(payment, 2)
-    return base_monthly_payment
+    monthly_payment = round(payment, 2)
+    total_cost = round(monthly_payment * loan_term_months, 2)
+    total_interest = round(total_cost - credit_amount, 2)
+    return monthly_payment, total_cost, total_interest
 
 
 def calculate_descending_rate(credit_amount, loan_term, interest_rate):
```

Comment:
I thought it over and maybe it would be better to modify fixed rate function, so it will return the list of the monthly payments?
So for the graph purposes it will show on frontend straight line... Ok, fixed rate is only temporary for 60 payments (first 5 years) for most of the banks, and then it switches back to descending rate or new round of fixed rate is negotiated

Reply:
For now it returns a list of the same value for each month. Will change it in a future update.

Reply:
Thanks resolving